ee/vellum_ee/workflows/display/tests/workflow_serialization/test_scheduled_trigger_serialization.py

```diff
@@ -0,0 +1,145 @@
+from contextlib import contextmanager
+import json
+import os
+import tempfile
+from unittest.mock import patch
+from uuid import UUID
+from typing import Iterator, Tuple
+
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.inputs.base import BaseInputs
+from vellum.workflows.nodes.bases.base import BaseNode
+from vellum.workflows.state.base import BaseState
+from vellum.workflows.triggers.base import _get_trigger_path_to_id_mapping
+from vellum.workflows.triggers.schedule import ScheduleTrigger
+from vellum_ee.workflows.display.workflows.get_vellum_workflow_display_class import get_workflow_display
+
+
+@contextmanager
+def mock_metadata_json(trigger_class_name: str, trigger_id: UUID) -> Iterator[Tuple[str, UUID]]:
```

Comment:
[oos] can avoid the additional nesting by using `pytest.fixture`'s not worth prioritizing until post tracer bullet